docs/course_materials/exercises/test_exercises/03_Variables_and_datatypes/test_homework_strings.py

```diff
@@ -0,0 +1,113 @@
+"""
+Automatically check student's homework
+----------------------------------------
+The tests will check:
+ 1. The content of the exercise file
+ 2. The execution and output of the exercise file
+ 3. The output with different inputs
+
+Guidelines:
+- You have to specify the dir Student's exercise files should be placed
+in the same dir as this test.
+- Test names are in the form: test_<exercise>.py
+- Before each test should be the appropriate fixture: @pytest.mark.<topic>
+"""
+import pytest
+
+
+@pytest.mark.strings
+def test_names(helpers):
+    helpers.student_file = helpers.home_dir + 'names.py'
```

Comment:
If you always attach the helpers.home_dir maybe you can create in the helpers a function for example 
def get_student_file(file)
   returns  home_dir + file